rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -25,19 +27,23 @@ class GameplayNode(Node):
 
     def __init__(self, play_selector: situation.IPlaySelector, world_state: Optional[rc.WorldState] = None) -> None:
         rclpy.init()
-        super().__init__('minimal_subscriber')
+        super().__init__('gameplay_node')
         self.world_state_sub = self.create_subscription(msg.WorldState, '/vision_filter/world_state', self.create_partial_world_state, 10)
         self.field_dimenstions = self.create_subscription(msg.FieldDimensions, '/config/field_dimensions', self.create_field, 10)
         self.game_info = self.create_subscription(msg.GameState, '/referee/game_state', self.create_game_info, 10)
         for i in range(NUM_ROBOTS):
             self.game_state_sub = self.create_subscription(msg.RobotStatus, '/radio/robot_status/robot_'+str(i), self.create_partial_robots, 10)
-        
+
         self.world_state = world_state
         self.partial_world_state: conv.PartialWorldState = None
         self.game_info: rc.GameInfo = None
         self.field: rc.Field = None
         self.robot_statuses: List[conv.RobotStatus] = []
 
+        self.global_parameter_client = GlobalParameterClient(
+            self, '/global_parameter_server')
+        local_parameters.register_parameters(self)
```

Comment:
So how would like an stp or action get access to a parameter?

Reply:
I tested this on my marking code. Stp gets params the same way as gameplay_node does:
```
import stp.global_parameters as global_parameters
[...]
def tick(...)
    print(global_parameters.soccer.robot.max_kick_speed)
```

Reply:
The local params also work like the testing, e.g. in `skill/mark.py`
```
sag_dist_param = Param("mark_sag_dist", RobotConstants.RADIUS * 0.5, Param.PARAMETER_DOUBLE)
[...]
mark_pos = opp_pos + mark_dir * (2 * RobotConstants.RADIUS + sag_dist_param.value)
```
This is slightly annoying because you have to reference the value with `<param_name>.value` rather than PARAM_NAME, as you would with a normal constant. Unless I'm missing something @kylestach ?